apkid/rules/elf/obfuscators.yara

```diff
@@ -215,3 +215,42 @@ rule avdobfuscator : obfuscator
     any of them and is_elf
 }
 
+rule arxan_native_arm : obfuscator
+{
+    meta:
+        description = "Arxan"
+        url         = "https://www.arxan.com/resources/technology/app-code-obfuscation"
+        sample      = "5bbb241d41c4150798b5800e62afcb6f49e05755d2fd89c7a9f7e356609c9012"
+        author      = "Eduardo Novella"
+
+    strings:
+        $a = {
```

Comment:
What is this from? The decoder?

Reply:
Prolog breakages

Reply:
Could you elaborate on this just a bit in a 1-2 line comment? I just want to capture hard-earned knowledge you might have in your head.

Reply:
Done